packages/components/src/sortTabs/SortTab.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import {
+  Text,
+  StyleSheet,
+  Icon,
+  Hoverable,
+} from '@kiwicom/universal-components';
+
+import TouchableWithoutFeedback from '../TouchableWithoutFeedback';
+import type { SortIconTypes, SortTypes } from './SortTabs';
+
+export type Props = {|
+  +label: string,
+  +value: SortTypes,
+  +first: boolean,
+  +last: boolean,
+  +active: boolean,
+  +icon: SortIconTypes,
+  +onPress: (value: SortTypes) => void,
```

Comment:
You don't need to write here `value:`. So 
```
+onPress: SortTypes => void,
```
would be enough

Reply:
Done!